bgpd/bgp_vty.c

```diff
@@ -7613,6 +7628,11 @@ DEFPY(bgp_imexport_vrf, bgp_imexport_vrf_cmd,
 		return CMD_WARNING;
 	}
 
+	if (strcmp(import_name, "route-map") == 0) {
```

Comment:
can't those two commands be merged into something like?:
`import vrf VIEWVRFNAME$import_name [route-map RMAP$rmap_str]`

Reply:
They could (and probably eventually should) but this fix is to stop some breakage in the existing cli.  I have a different ticket assigned to me to combine them but for the time being, making the current commands work correctly seems to make sense.  There are users using the current methods and if I just change to a combined command, I would be breaking them.